webpack/assets/javascripts/bundle_novnc.js

```diff
@@ -0,0 +1,89 @@
+import RFB from '@novnc/novnc/core/rfb';
+
+let rfb;
+
+function sendCtrlAltDel() {
+  rfb.sendCtrlAltDel();
+  return false;
+}
+
+function showStatus(state, message) {
+  let level;
+  const status = $('#noVNC_status');
+  const ctrlAltDeleteButton = $('#ctrlAltDelButton');
+
+  switch (state) {
+    case 'failed':
+      level = 'danger';
+      break;
+    case 'fatal':
+      level = 'danger';
+      break;
+    case 'normal':
+      level = 'success';
+      break;
+    case 'disconnected':
+      level = 'default';
+      break;
+    default:
+      level = 'warning';
+      break;
+  }
+
+  ctrlAltDeleteButton.prop('disabled', state !== 'normal');
+
+  if (typeof message !== 'undefined') {
+    status.attr('class', `col-md-12 label label-${level}`);
+    status.html(message);
+  }
+}
+
+function securityFailed(e) {
+  let msg = '';
+  if ('reason' in e.detail) {
+    msg = `New connection has been rejected with reason: ${e.detail.reason}`;
+  } else {
+    msg = 'New connection has been rejected';
+  }
+  showStatus('fatal', msg);
+}
+
+function disconnectFinished() {
+  showStatus('failed', 'Disconnected');
+}
+
+function connectFinished() {
+  showStatus('normal', 'Connected');
+}
+
+$(document).on('ContentLoad', () => {
```

Comment:
this will now execute on every page, you should probably exit this function early once there is no vnc in the page?

Reply:
Hi @ohadlevy , thank you for the review. Is was wondering, I actually did the check on the line 62, if there is no id noVNC_screen, I don't execute anything else. Is there a better way? Should I check on path rather than on content? The best path is probably to make this or use a react component; but I'd like to descope this for now and make a separate ticket for this as I'd like to make an RFC on the whole console experience beforehand; if that's okay for you :-)